assembl/static2/tests/unit/components/svg/taxonomy/Flag.spec.jsx

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Flag from '../../../../../js/app/components/svg/taxonomy/Flag';
+
+describe('Flag component', () => {
+  it('should render a colored flag svg', () => {
```

Comment:
@cedricmessiant with @MilanBarande we discussed a lot about snapshot test vs regular "given, when, then" unit test. I think we kind of loose an explicit information: let's say you want to test "the flag should be green" and the "flag should be red", the test with snapshot would look exactly the same, isn't it ? Well this is just food for thought nothing to blame here 👍